student.rb

```diff
@@ -1,11 +1,18 @@
 require './person'
 
 class Student < Person
+  attr_reader :classroom
+
   def initialize(age, classroom, name = 'Unknown', parent_permission: true)
     super(age, name, parent_permission)
     @classroom = classroom
   end
 
+  def classroom=(classroom)
+    @classroom = classroom
+    classroom.students.push(self) unless classroom.students.include?(self)
+  end
```

Comment:
- [ ] Great work on implementing the `belongs_to` relationship between students and class :+1: